index.css

```diff
@@ -0,0 +1,229 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-size: 14px;
+  font-family: arial, sans-serif;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  background-color: #fff;
+  position: relative;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row-reverse;
+  height: 8%;
+}
+
+.navbar ul {
+  font-size: 13px;
+  margin: 3px 10px 0px 0px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+li {
+  display: inline;
+  margin-right: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: #000000;
+}
+
+img {
+  height: 24px;
+}
+
+.title {
+  height: 26%;
```

Comment:
1. Don't use relative height at any place. This will break the design as screen size will change.
2. Use absolute values for defining height.
3. You can use min and max height property if you want to clamp the height b/w certain values.
4. Example: This search bar was looking fine on your screen height but it breaks on mine because you're using relative height.
<img width="685" alt="image" src="https://github.com/saurabh-rajniwal/home-page/assets/34623291/78db03e6-e37e-43c1-bfb7-926dfe869731">


Check at all places in your code.